lib/frontend/src/frontend.rs

```diff
@@ -41,7 +41,7 @@ where
     /// Source location to assign to all new instructions.
     srcloc: ir::SourceLoc,
 
-    builder: &'a mut ILBuilder<Variable>,
+    builder: &'a mut FunctionBuilderContext<Variable>,
```

Comment:
I see you updated an `il_builder` variable below; we can also rename the `builder` field here.